sklearn/_fast_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+from sklearn.utils.testing import assert_raises_regex
+from sklearn.datasets import make_classification, make_regression
+from sklearn.utils.estimator_checks import check_estimator
+
+from sklearn.ensemble import FastGradientBoostingClassifier
+from sklearn.ensemble import FastGradientBoostingRegressor
+
+
+X_classification, y_classification = make_classification(random_state=0)
+X_regression, y_regression = make_regression(random_state=0)
+
+
+@pytest.mark.parametrize('GradientBoosting, X, y', [
+    (FastGradientBoostingClassifier, X_classification, y_classification),
+    (FastGradientBoostingRegressor, X_regression, y_regression)
+])
+def test_init_parameters_validation(GradientBoosting, X, y):
+
+    assert_raises_regex(
```

Comment:
I think we are migrating to the `pytest.raises(match=...)` context manager syntax.